wagtailimporter/serializer.py

```diff
@@ -70,11 +69,11 @@ class JSONEncoder(json.JSONEncoder):
     Extension of the JSON encoder that knows how to encode the YAML objects
     """
 
-    def default(self, obj):  # pylint:disable=method-hidden
-        if isinstance(obj, JSONSerializable):
-            return obj.__to_json__()
+    def default(self, o):  # pylint:disable=method-hidden
```

Comment:
Prefer things to have descriptive names. `obj` is superior to `o`.

Reply:
I agree, but then pylint complains about changing argument names compared with super().default(), which uses `o`. I can disable the warning instead of renaming the argument.

Reply:
Hrm. Damned either way. Follow Python's lead I guess.